sol-tests/party/PartyGovernanceNFT.t.sol

```diff
@@ -398,6 +468,33 @@ contract PartyGovernanceNFTTest is Test, TestUtils {
     }
 }
 
+contract DummyCrowdfund {
+    uint256 public totalContributions;
+    bytes32 public governanceOptsHash;
+
+    function setGovernanceOptsHash(Crowdfund.FixedGovernanceOpts memory opts) external {
+        governanceOptsHash = _hashFixedGovernanceOpts(opts);
+    }
+
+    function _hashFixedGovernanceOpts(
```

Comment:
Nitpick, but I hate seeing this copy-pasted all over. Maybe a use case for a free function?